### ISSUE TITLE
XcelEDAReportingandQAQC failing on 3.6.0 and CI issue with 3.5.1

### ISSUE THREAD

**Issue by DavidGoldwasser** [NREL/openstudio-common-measures-gem#140]
On Mac 3.6.0 OpenStudio I see this error

```XcelEDAReportingandQAQC_Test#test_example_model:
NoMethodError: undefined method `otherFuelTotalEndUses' for #<OpenStudio::SqlFile:0x00007f8a2d35aef8>
    /Users/dgoldwas/Documents/github/nrel/openstudio-common-measures-gem/lib/measures/XcelEDAReportingandQAQC/resources/CreateResults.rb:348:in `create_results'
    /Users/dgoldwas/Documents/github/nrel/openstudio-common-measures-gem/lib/measures/XcelEDAReportingandQAQC/measure.rb:166:in `run'
    /Users/dgoldwas/Documents/github/nrel/openstudio-common-measures-gem/lib/measures/XcelEDAReportingandQAQC/tests/XcelEDAReportingandQAQC_Test.rb:231:in `test_example_model'

**Comment by DavidGoldwasser** [comment]
The CI for PR's that is using some version of 3.5.1 shows the error below. On mac locally measure runs fine in 3.5.1 and 3.5.0.

Failure: test_example_model(Minitest::Result) [/srv/jenkins/git/openstudio-common-measures-gem/PR-135/1/lib/measures/generic_qaqc/resources/CreateResults.rb:942]: NoMethodError: undefined method >' for nil:NilClass /srv/jenkins/git/openstudio-common-measures-gem/PR-135/1/lib/measures/generic_qaqc/resources/CreateResults.rb:942:in block in create_results' /srv/jenkins/git/openstudio-common-measures-gem/PR-135/1/lib/measures/generic_qaqc/resources/CreateResults.rb:931:in each' /srv/jenkins/git/openstudio-common-measures-gem/PR-135/1/lib/measures/generic_qaqc/resources/CreateResults.rb:931:in create_results' /srv/jenkins/git/openstudio-common-measures-gem/PR-135/1/lib/measures/XcelEDAReportingandQAQC/measure.rb:166:in run' /srv/jenkins/git/openstudio-common-measures-gem/PR-135/1/lib/measures/XcelEDAReportingandQAQC/tests/XcelEDAReportingandQAQC_Test.rb:231:in test_example_model'

@tijcolem when you update CI to 3.6.0 this may go away, but the one above will show up. I'm wondering if the 3.5.1 CI is develop which may have more changes than were in the 3.5.1 release?

**Comment by joseph-robertson** [comment]
I believe `otherFuelTotalEndUses` was deprecated (and then removed altogether) in https://github.com/NREL/OpenStudio/pull/4725. Should be replaced with the sum of new `otherFuel1TotalEndUses` and `otherFuel2TotalEndUses` methods? @kflemin might have better guidance here.

**Comment by kflemin** [comment]
@DavidGoldwasser, @joseph-robertson, see: https://github.com/NREL/openstudio-common-measures-gem/pull/142

**Comment by joseph-robertson** [comment]
Oh cool, looks like @kflemin beat me to it! 👍

**Comment by DavidGoldwasser** [comment]
Thanks @kflemin .  @joseph-robertson the CI now shows the fuel error (bellow), so I think if you merge Katherine's branch that measure should pass, however I deleted `CreateResults.rb` in the `generic_qaqc` measure since it wasn't used. Also measure xml's will need to be updated.

`
Failure: test_example_model(Minitest::Result) [/srv/jenkins/git/openstudio-common-measures-gem/PR-144/1/lib/measures/XcelEDAReportingandQAQC/resources/CreateResults.rb:348]: NoMethodError: undefined method `otherFuelTotalEndUses' for #<OpenStudio::SqlFile:0x00005592f9410938> /srv/jenkins/git/openstudio-common-measures-gem/PR-144/1/lib/measures/XcelEDAReportingandQAQC/resources/CreateResults.rb:348:in `create_results' /srv/jenkins/git/openstudio-common-measures-gem/PR-144/1/lib/measures/XcelEDAReportingandQAQC/measure.rb:166:in `run' /srv/jenkins/git/openstudio-common-measures-gem/PR-144/1/lib/measures/XcelEDAReportingandQAQC/tests/XcelEDAReportingandQAQC_Test.rb:231:in `test_example_model'
`

There is a new issue now with a different measure `GemEnvironmentReport`. I'll reference this issue in one one

**Comment by DavidGoldwasser** [comment]
Here the error XcelEDAReportingandQAQC sees now.

`
XcelEDAReportingandQAQC_Test#test_example_model:
NameError: uninitialized constant #<Class:#<OpenStudio::SqlFile:0x00007fc2f144f410>>::OtherFuel2TotalEndUses
    eval:368:in `const_missing'
    (eval):1:in `block in create_results'
    /Users/dgoldwas/Documents/github/nrel/openstudio-common-measures-gem/lib/measures/XcelEDAReportingandQAQC/resources/CreateResults.rb:351:in `instance_eval'
    /Users/dgoldwas/Documents/github/nrel/openstudio-common-measures-gem/lib/measures/XcelEDAReportingandQAQC/resources/CreateResults.rb:351:in `block in create_results'
    /Users/dgoldwas/Documents/github/nrel/openstudio-common-measures-gem/lib/measures/XcelEDAReportingandQAQC/resources/CreateResults.rb:350:in `each'
    /Users/dgoldwas/Documents/github/nrel/openstudio-common-measures-gem/lib/measures/XcelEDAReportingandQAQC/resources/CreateResults.rb:350:in `create_results'
    /Users/dgoldwas/Documents/github/nrel/openstudio-common-measures-gem/lib/measures/XcelEDAReportingandQAQC/measure.rb:166:in `run'
    /Users/dgoldwas/Documents/github/nrel/openstudio-common-measures-gem/lib/measures/XcelEDAReportingandQAQC/tests/XcelEDAReportingandQAQC_Test.rb:231:in `test_example_model'

2 runs, 10 assertions, 0 failures, 1 errors, 1 skips

**Comment by DavidGoldwasser** [comment]
Fixed with #157
